docs/next.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     ignoreDuringBuilds: true,
   },
   // Avoid conflicts with the other Next.js apps hosted under https://mui.com/
-  assetPrefix: isDeployment ? '/x' : '',
+  assetPrefix: isDeployment ? '/x' : undefined,
```

Comment:
To avoid next config validation warning:
```
  {
    "instancePath": "/assetPrefix",
    "schemaPath": "#/properties/assetPrefix/minLength",
    "keyword": "minLength",
    "params": {
      "limit": 1
    },
    "message": "must NOT have fewer than 1 characters"
  }
```